src/nurse/management/commands/_notifications.py

```diff
@@ -9,15 +11,33 @@ def get_client():
 
 
 def notify():
-    notification_body = {
-        "contents": {
-            "fr": "Une ordonnance est sur le point d'expirer, "
-            "ouvrez l'application pour la consulter.",
-            "en": "A prescription is about to expire open the app to review.",
-        },
-        "included_segments": ["Subscribed Users"],
-        "name": "PRESCRIPTION EXPIRE SOON",
-    }
+    prescriptions = Prescription.objects.expiring_soon()
+
+    for prescription in prescriptions:
+        patient = prescription.patient
+        nurse = patient.nurse_set.first()
+        if nurse and nurse.user:
+
+            filter_args = {"user": nurse.user}
+            subscription_id_list = UserOneSignalProfile.objects.filter(
+                **filter_args
+            ).values_list("subscription_id", flat=True)
+            subscription_id = subscription_id_list.first()
+            if subscription_id:
+                contents_dict = {
+                    "fr": (
+                        "Une ordonnance est sur le point d'expirer, "
+                        "ouvrez l'application pour la consulter."
+                    ),
+                    "en": (
+                        "A prescription is about to expire " "open the app to review."
+                    ),
```

Comment:
This may grow big over time, maybe move it outside the loop to the same file or even in a dedicated file.
A positive side effect of doing it would be that we can import it in our unit tests to compare the strings in the sent notification

Reply:
Done in the same file